daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -514,12 +516,46 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments) {
     requireSubcommand()
   }
 
+  // Encode EXI Subcommand Options
+  object encodeEXI extends scallop.Subcommand("encodeEXI") {
+    banner("""|Usage: daffodil encodeEXI [-s <schema>] [-o <output>] [infile]
```

Comment:
Thoughts on having a single subcommand `exi` that can both encode and decode? For example, the `base64` command encodes by default and decodes with the `-d` option. Maybe something similar?

I almost wonder if this wants to be a separate utility, e.g. `dfdlexi`? Might not be worth the effort.

Reply:
Yes, I think it's better to have a single subcommand `exi` which normally encodes XML to EXI and optionally decodes EXI to XML with the `-d` or `--decode` options.